libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -182,6 +187,11 @@ public boolean removeAttachmentListener(AttachmentListener listener) {
     return attachmentListeners.remove(listener);
   }
 
+  @VisibleForTesting
+  void setBuildVersion(int buildVersion) {
```

Comment:
we don't actually need this setter method and `buildVersion` variable. we could instead stub `isLollipopOrHigher()` method instead.